src/routes/(site)/s23positions/data.ts

```diff
@@ -0,0 +1,93 @@
+import type { ListItem } from '$lib/components/recursive-ul/types';
+import { TEAMS } from '$lib/public/board/data';
+import type { ClubPosition } from './position';
+
+/**
+ * The tools that are used in the club.
+ *
+ * @remarks This is a map of tool names to their reasons for being used.
+ */
+export const TOOLS = {
+  'Google Drive/Docs': 'To collaborate on presentations, documents, and spreadsheets.',
+  Discord: 'Internal team communication, assistance for student questions',
+  Figma: 'Create marketing materials and teaching students how to use it',
+  GitHub: 'To collaborate on code, manage projects, and teach students how to use it',
+  'Social media': 'Discord, Instagram, LinkedIn, and YouTube',
+  'Google Colab/Juptyer Notebook': 'Cloud development environment for Python',
+};
+
+function li(html: string, children?: ListItem[]): ListItem {
+  return { html, children };
+}
+
+export const POSITIONS: ClubPosition<keyof typeof TOOLS>[] = [
```

Comment:
Finish filling out the rest of the data for the available positions.
- [x] AI Officer
- [x] Algo Officer
- [ ] Design Officer
- [x] Dev Officer
- [ ] Game Dev Officer
- [ ] Marketing Officer

Reply:
- [x] AI Officer
- [x] Algo Officer
- [x] Design Officer
- [x] Dev Officer
- [x] Game Dev Officer
- [x] Marketing Officer

Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/754/commits/3ef17687c40e597e980d61db5f9d67035076c3e4>.